_forms/stance.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: form
+title: "Stance"
+title_mark: Stance
+published_date: 2025-11-06
+refactored_date: 2025-11-07
+authors:
+  - michael-basil
+  - kyle-ingersoll
+excerpt: "A Stance is the personal expression of one's FEBI™ energy archetypes through breath, tone, timing, and attention."
+---
+
+## Intention
+
+A Stance is the personal expression of one's FEBI™ energy archetypes through breath, tone, timing, and attention. In short, one's attitude during the moments that matter. Different people have different Stances, and Stances can be shifted. Apply both the Platnum and Golden Rules when shifting Stances.
+
+## Experience
+
+You resonate/vibe with the Other, truely present and connected with them. You then lead them by example in the conversational direction you want them to go.
```

Comment:
resonate/vibe with the people int he group and overall field.

Sometimes it is leading by example, but always not where "want them to go", rather understanding their energy capabilities (learning as we go if less familiar) and inviting for mutually beneficial and well timed shifts